### DISCUSSION TITLE
How to keep the content of the insert node after the choice is changed?

### DISCUSSION THREAD

**Discussion by fovj** [L3MON4D3/LuaSnip#956] [Q&A] [answered]
I had this choice nodes config:
```lua
  s(
    "ii",
    c(1, {
      fmta("One: <>", {
        i(1, ""),
      }),
      fmta("Two: <>", {
        i(1, ""),
      }),
    })
  ),

```

After the choice is changed, the content of the insert node will not be retained

e.g. `One: abc`  -> `Two:`

If want to keep the content of the insert node after changing the choice, how to config it?

e.g. `One: abc`  -> `Two: abc`

**Comment by L3MON4D3** [comment, marked as answer]
`restoreNode` :wink:

**Reply by fovj** [reply]
![nice-smack](https://github.com/L3MON4D3/LuaSnip/assets/4456413/1e75a804-d521-4cb3-b223-a47a257bdfac)
